cmake/FindADIOS.cmake

```diff
@@ -178,6 +179,9 @@ if(ADIOS_FOUND)
     endforeach()
     # we could append ${CMAKE_PREFIX_PATH} now but that is not really necessary
 
+    # determine whether found library is serial only
```

Comment:
I think we can only termine:
- is serial *and* parallel
- is serial only

Reply:
Alright, the vocabulary in this comment is not sufficient. This determines if the library *links*  as serial only.
The library itself might be serial + parallel.

Reply:
ok cool.

want I wanted to say is just: if you build serial, the `lib/` dir will contain a regular and a `_nompi` version, as far as I remember.

If you check that only the noMPI is linked this is fine, but you might want to prefer it someway during search and validate it with two adios installations (/w and /wo MPI) to make sure.